less/pageLevelProgressIndicator.less

```diff
@@ -8,19 +8,22 @@
     border: 0.0625rem solid @black;
     border-radius: 50px;
     overflow: hidden;
+    padding: 0.1rem;
```

Comment:
How about setting the padding at `0.0625rem` for consistency with the border? It's slightly more subtle and makes the overall progress bar less chunky.